components/BCRegistriesAccount.vue

```diff
@@ -1,250 +1,247 @@
-            </p>
-          </v-card-text>
-        </v-card>
-      </v-card>
+      <div v-if="!loggedIn" class="mt-5 mr-4 d-sm-flex flex-wrap justify-center">
+        <SbcSigninButton />
 
-      <div class="payment pt-7 pb-7 mr-10">
-        <v-row no-gutters class="ma-0 pa-0">
-          <v-col cols="2"></v-col>
-          <v-col cols="8" class="payment-info-bold">
-            <v-icon medium color="#38598a">mdi-currency-usd-circle</v-icon>
-            <span style="padding-left: 10px">Multiple payment types:</span>
-          </v-col>
-          <v-col cols="2"> </v-col>
-        </v-row>
-
-        <v-row no-gutters class="ma-0 pa-0">
-          <v-col cols="3"></v-col>
-          <v-col cols="6" class="payment-info" style="max-width: 850px">
-            The BC Registries Account supports multiple payment options
-            depending on the account and transaction type, including card,
-            online banking, BC OnLine, and pre-authorized debit (PAD).
-          </v-col>
-          <v-col cols="3"></v-col>
-        </v-row>
-
-        <v-card
-          flat
-          tile
-          class="menu-card-button justify-center d-flex flex-wrap"
+        <v-btn
+          large
+          class="button-blue-on-white"
+          :href="$config.regAccountCreateURL"
         >
-          <v-card flat tile class="pr-3 bg-gray2-5">
-            <SbcSigninButton />
-          </v-card>
-
-          <v-card flat tile class="pl-3 bg-gray2-5">
-            <v-btn
-              class="button-gold"
-              style="height: 40px"
-              :href="$config.regAccountCreateURL"
-            >
-              Create a BC Registries Account
-            </v-btn>
-          </v-card>
-        </v-card>
+          Create a BC Registries Account
+        </v-btn>
+      </div>
+
+      <div v-if="loggedIn" class="text-center mt-5 mr-4">
+        <v-btn
+          large
+          class="button-white-on-blue"
+          href="/dashboard"
+        >
+          Go to BC Registries Dashboard
+        </v-btn>
       </div>
-    </v-container>
-  </div>
+    </div>
+  </v-container>
 </template>
 
 <script>
+import { SessionStorageKeys } from 'sbc-common-components/src/util/constants'
 import SbcSigninButton from '~/components/SbcSigninButton.vue'
+
 export default {
   components: {
     SbcSigninButton,
   },
+  computed: {
+    loggedIn() {
+      if (sessionStorage.getItem(SessionStorageKeys.KeyCloakToken)) {
+        return true
+      }
+      return false
+    },
+  },
 }
 </script>
 
 <style lang="scss" scoped>
 @import '../assets/scss/theme.scss';
 
-a {
+.link {
   color: $BCgovLink !important;
-  font-size: 1rem;
   font-weight: bold;
-  text-decoration: underline;
 }
 
-.bg-gray1 {
-  background-color: $gray1;
+img {
+  margin-bottom: -6px;
 }
 
-.bg-gray2-5 {
-  background-color: $gray2-5;
+.v-btn,
+::v-deep .sbc-signin-button {
+  margin: 0.5rem;
+}
+
+// "sm" breakpoint
+@media (max-width: 599px) {
```

Comment:
Up to this breakpoint, buttons are full-width. After this, the div containing the buttons uses flex to display them.